vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -325,12 +333,12 @@ class LEDGER_EXPORT Ledger {
   virtual void FetchBalance(FetchBalanceCallback callback) = 0;
 
   virtual void GetExternalWallet(const std::string& wallet_type,
-                                 ExternalWalletCallback callback) = 0;
+                                 GetExternalWalletCallback) = 0;
 
-  virtual void ExternalWalletAuthorization(
+  virtual void ConnectExternalWallet(
       const std::string& wallet_type,
       const base::flat_map<std::string, std::string>& args,
-      ExternalWalletAuthorizationCallback callback) = 0;
+      ConnectExternalWalletCallback) = 0;
```

Comment:
Actually, I started removing them quite a while ago, since it doesn't add anything to what we already know: `ConnectExternalWalletCallback` is a callback. 🙂 Same with `mojom::Result result`. It just feels unnecessary to give them a name that's otherwise trivial.